yb-voyager/cmd/live_migration_integration_test.go

```diff
@@ -1098,3 +1098,193 @@ END $$;`,
 	testutils.FatalIfError(t, err, "Cutover command failed")
 
 }
+
+func TestLiveMigrationWithUniqueKeyConflictWithUniqueIndexOnlyOnLeafPartitions(t *testing.T) {
+	ctx := context.Background()
+
+	// Create a temporary export directory.
+	exportDir = testutils.CreateTempExportDir()
+	// defer testutils.RemoveTempExportDir(exportDir)
+
+	createSchemaSQL := `CREATE SCHEMA IF NOT EXISTS test_schema;`
+
+	createTableWithUniqueIndexOnlyOnSomeLeafPartitionsSQL := `
+	CREATE TABLE test_schema.test_partitions (
+		id int,
+		name TEXT,
+		region TEXT,
+		branch TEXT,
+		PRIMARY KEY(id, region)
+	) PARTITION BY LIST (region);
+
+	CREATE TABLE test_schema.test_partitions_part1 PARTITION OF test_schema.test_partitions FOR VALUES IN ('London');
```

Comment:
As we were discussing, shall we also add a test case where the leaf partitions are in a different schema?